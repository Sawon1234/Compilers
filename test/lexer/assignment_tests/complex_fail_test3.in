fun not(b : Boolean) : Boolean
{
  if (b)
  {
    return false;
  }
  return true;
}