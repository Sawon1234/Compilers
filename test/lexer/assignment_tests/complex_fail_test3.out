fun name ( name : Name ) : Name { if ( name ) { return true ; } return true ; }